### DISCUSSION TITLE
Pull Request "Files changed" pagination

### DISCUSSION THREAD

**Discussion by Kaspik** [community/community#78271] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When opening a PR, and trying to check the changes in "Files changed" tab, in bigger PRs on in a PRs with a lot of updated files (for example snapshot test images), it's basically impossible to review it or at least approve it.

It would be helpful if Github would load part of the files changes and there would be "Load more" button. During the review, I would mark all the visible files as "Viewed", so on "Load more" new content would be added / appended.

**Comment by cipherleaf** [comment]
Please make this happen. On pull request with lots of file, the changes on the files on the bottom are not even loaded and you have to click to load it. On big PRs with lots of files this 'cause the page slower and takes forever to load. Please add pagination 🙇🏼
